torch/_inductor/fx_passes/post_grad.py

```diff
@@ -102,6 +102,11 @@ def post_grad_passes(gm: torch.fx.GraphModule, is_inference: bool):
             post_grad_custom_pre_pass
         )
 
+    if torch._C._has_mkldnn:
```

Comment:
Should we move the config check here?  Will this pass do anything if `enable_grouped_gemm_template=False`?

Reply:
No, this pass do nothing if `enable_grouped_gemm_template=False` and move this config check here.